.github/actionlint.yaml

```diff
@@ -0,0 +1,3 @@
+self-hosted-runner:
```

Comment:
Why is this file named "actionlint" rather than "e2e"?

Reply:
because it's a config file for the actionlint workflow. This is needed so that actionlint considers `ubuntu-gpu` a valid runner type

Reply:
it's a copy of the same file in `instructlab/instructlab`

Reply:
Ah! I did not make that connection. Silly me.